hbase-client/src/main/java/org/apache/omid/transaction/TTable.java

```diff
@@ -132,7 +133,7 @@ public Result get(Transaction tx, final Get get) throws IOException {
 
         HBaseTransaction transaction = enforceHBaseTransactionAsParam(tx);
 
-        final long readTimestamp = transaction.getStartTimestamp();
```

Comment:
Please add external comments about read and write timestamps, and how they relate to visibility levels.